src/App.js

```diff
@@ -1,39 +1,95 @@
 import "bootstrap/dist/css/bootstrap.min.css";
-import React from "react";
+import React, {useState} from "react";
 import {
   Welcome,
-  NavMenu,
+  Photos,
+  Nav2,
   Story,
-  StoryImage,
   Faq,
   Email,
   COVID,
   Footer,
 } from "./components/index";
 import styled from "styled-components";
 import GlobalStyle from "./globalStyles";
+import {device} from "./device";
+import {background2} from "../src/img/index";
+import useLocalStorage from "react-use-localstorage";
+import {divider} from "./img/index";
 
 function App() {
+  // need to update the following to use LoginContext
+  const [password, setPassword] = useState("");
+  const [isAuthenticated, setIsAuthenticated] = useState();
+  const [localAuth, setLocalAuth] = useLocalStorage("isAuth", false);
+
+  const handleChange = (e) => {
+    const input = e.target.value;
+    console.log("input: ", input);
+    setPassword(input);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log("Submitted");
+    if (password !== "123") {
```

Comment:
This looks like something that should be configurable

Reply:
Having some trouble here. Tried using REACT_APP_ but that didn't work. Installed react-dotenv to try to grab it off of env as outlined here: https://www.npmjs.com/package/react-dotenv but still came back undefined.